packages/shared/lib/units.ts

```diff
@@ -1,20 +1,35 @@
-import { Unit } from '@iota/unit-converter'
 import Big from 'big.js'
+import BigNumber from 'bignumber.js'
 import { getCurrencyPosition, formatNumber } from 'shared/lib/currency'
 
 // Set this to avoid small numbers switching to exponential format
 Big.NE = -20
 
+export enum Unit {
```

Comment:
Since this is for IOTA, should we name this `IotaUnit` rather than just `Unit`?

Reply:
It is going to be used for any Metric amount, but its not needed at the moment and the refactor of the units is a can of worms. I will open an issue to refactor the units and formatters into the assets module.